cas-server-support-jpa-util/src/main/java/org/jasig/cas/util/jpa/ZonedDateTimeConverter.java

```diff
@@ -0,0 +1,28 @@
+package org.jasig.cas.util.jpa;
+
+import org.jasig.cas.util.DateTimeUtils;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+import java.time.ZonedDateTime;
+import java.util.Date;
+
+/**
+ * Converts ZonedDateTime <-> Date.
+ * @author tduehr
+ * @since 4.3.0
+ */
+@Converter(autoApply = true)
+public class ZonedDateTimeConverter implements AttributeConverter<ZonedDateTime, Date> {
```

Comment:
If the purpose of the module is to convert `ZonedDateTime` to something the schema can understand, we should then remove this module and make sure all the ticket api is considered as long.